contracts/contracts/interfaces/IMACI.sol

```diff
@@ -3,31 +3,31 @@ pragma solidity ^0.8.10;
 
 import { AccQueue } from "../trees/AccQueue.sol";
 
-/// @title IMACI interface
+/// @title MACI interface
```

Comment:
the title should be IMACI I believe 
then can add a description saying "MACI interface"